library/core/src/hint.rs

```diff
@@ -223,12 +223,12 @@ pub fn spin_loop() {
 ///
 /// # When is this useful?
 ///
-/// First and foremost: `black_box` does _not_ guarantee any exact behavior and, in some cases, may
-/// do nothing at all. As such, it **must not be relied upon to control critical program behavior.**
+/// First and foremost: `black_box` does _not_ guarantee any exact behavior beyond behaving as the identity function
+/// and, in some cases, does nothing at all. As such, it **must not be relied upon to control critical program behavior.**
```

Comment:
Fwiw, I think this is a bit redundant because it's mentioned one line above https://github.com/rust-lang/rust/blob/6f7ca32ae7f5cd9159a986ca0c30b77420cce933/library/core/src/hint.rs#L219-L220

Reply:
@tgross35 Thanks for reviewing this PR. I've adjusted wording to be a bit less redundant.